docs/docs-book/src/introduction.md

```diff
@@ -1,11 +1,72 @@
-# What is Coffee? and Why we need it?
+# What is Coffee? and Why do we need it?
 
-TODO
+Coffee is a plugin manager for core lightning that supports the user
+and the developers while working and using the core lightning plugin
+architecture.
+
+> The plugin manager is under development so at the current moment
+only the code is the real docs and some of the things that are marked as
+unsupported in the docs may be already supported in the code
 
 ## Problems
 
+Core lightning has a very powerful plugin system that empowers
+everyone to write a plugin for core lightning to extend the functionality
+of it.
+
+In fact, some of the core features like the `pay` command in core lightning is
+itself a plugin.
+
+However, usually when you give the power to everyone to write their own
+way to write a plugin, this will start to fragment the ecosystem in a very
+bad way.
+
+In fact, currently, we had many ways to write a plugin in core lightning
+with different languages, like rust, go, java, kotlin, scala, dart, python
+and many others. And this can cause confusion around the community and
+the user now has a different way to install a plugin and different
+way to compile and install a plugin.
```

Comment:
```suggestion
and many others. This can lead to disarray in the community, as the user now has to follow a different procedure for installing a plugin and building it.
```